sound/soc/sof/mediatek/mt8186/mt8186.c

```diff
@@ -628,7 +629,134 @@ static const struct sof_dev_desc sof_of_mt8186_desc = {
 	.ops = &sof_mt8186_ops,
 };
 
+static void mt8188_adsp_dump(struct snd_sof_dev *sdev, u32 flags)
```

Comment:
I thinks it may keep the name "8168"  to consistent with other functions and also add dbg_dump callback in sof_mt8188_ops . maybe discuss with @tinghan-shen? Thanks.

Reply:
OK. I will rename the function, and add it to sof_mt8168_ops.